conanfile.py

```diff
@@ -0,0 +1,44 @@
+from conans import ConanFile, CMake, tools
+
+class DbPostgreSQLAdapterConan(ConanFile):
+    name = "DbPostgreSQLAdapter"
+    description = "Implementation of C++ DB adapter based on PostgreSQL"
+    url = "https://github.com/systelab/cpp-postgresql-db-adapter"
+    homepage = "https://github.com/systelab/cpp-postgresql-db-adapter"
+    author = "CSW <csw@werfen.com>"
+    topics = ("conan", "db", "sql", "postgreSQL", "adapter")
+    license = "MIT"
+    generators = "cmake_find_package"
+    settings = "os", "compiler", "build_type", "arch"
+    exports_sources = "*", "!*.yml", "!*.md", "!.gitattributes", "!.gitignore", "!LICENSE"
+
+    def configure(self):
+        self.options["libpq"].shared = True
+        self.options["libpq"].with_openssl = True
+
+    def requirements(self):
+        self.requires("DbAdapterInterface/0.0.0@systelab/testing")
+        self.requires("openssl/3.0.13@", override=True)
+        self.requires("libpq/15.4@")
+
+    def build_requirements(self):
+         self.build_requires("gtest/1.14.0@")
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure(source_folder=".")
+        cmake.build()
+
+    def imports(self):
```

Comment:
Shouldn't you be packaging the .h files inside an include folder?

Reply:
Yep, I completely forgot about it :( thanks